diff --git a/jsonapi_layout_builder.module b/jsonapi_layout_builder.module
index 8045eda..dee2866 100644
--- a/jsonapi_layout_builder.module
+++ b/jsonapi_layout_builder.module
@@ -4,7 +4,9 @@
  * @file
  * Contains jsonapi_layout_builder.module.
  */
-
+use Drupal\jsonapi_layout_builder\Layout\LayoutSettings;
+use Drupal\block_content\Entity\BlockContent;
+use Drupal\Core\Field\FieldStorageDefinitionInterface;
 use Drupal\Core\Entity\ContentEntityBase;
 use Drupal\Core\Entity\EntityInterface;
 use Drupal\Core\Routing\RouteMatchInterface;
@@ -48,7 +50,7 @@ function jsonapi_layout_builder_layout_alter(&$definitions) {
   foreach ($definitions as $id => $def) {
     /** @var \Drupal\Core\Layout\LayoutDefinition $def */
     $class = $def->getClass();
-    $def->setClass(\Drupal\jsonapi_layout_builder\Layout\LayoutSettings::class);
+    $def->setClass(LayoutSettings::class);
     $def->set('base_class', $class);
   }
 }
@@ -67,7 +69,7 @@ function jsonapi_layout_builder_entity_presave(EntityInterface $entity) {
       if (strpos($config['id'], 'inline_block:') === 0) {
         $block = \Drupal::service('entity.repository')
           ->loadEntityByUuid('block_content', $config['uuid']);
-        if (!$block instanceof \Drupal\block_content\Entity\BlockContent) {
+        if (!$block instanceof BlockContent) {
           continue;
         }
         $components[$uuid] = $block->id();
@@ -125,7 +127,7 @@ function __jsonapi_layout_builder_add_components_field($entity_type_id, $bundle,
         'field_name' => $field_name,
         'type' => 'entity_reference',
         'locked' => TRUE,
-        'cardinality' => \Drupal\Core\Field\FieldStorageDefinitionInterface::CARDINALITY_UNLIMITED,
+        'cardinality' => FieldStorageDefinitionInterface::CARDINALITY_UNLIMITED,
         'settings' => [
           'target_type' => 'block_content',
         ],
diff --git a/src/LayoutBuilderLoaderManager.php b/src/LayoutBuilderLoaderManager.php
index f5b8a69..5307453 100644
--- a/src/LayoutBuilderLoaderManager.php
+++ b/src/LayoutBuilderLoaderManager.php
@@ -2,6 +2,7 @@
 
 namespace Drupal\jsonapi_layout_builder;
 
+use Symfony\Component\Routing\Route;
 use Drupal\Core\Entity\ContentEntityBase;
 use Drupal\Core\Entity\FieldableEntityInterface;
 use Drupal\Core\Plugin\Context\Context;
@@ -62,7 +63,7 @@ class LayoutBuilderLoaderManager {
     if ($this->route === NULL) {
       $this->route = \Drupal::routeMatch()->getRouteObject();
     }
-    if (!$this->route instanceof \Symfony\Component\Routing\Route || !$this->route->hasDefault('_is_jsonapi')) {
+    if (!$this->route instanceof Route || !$this->route->hasDefault('_is_jsonapi')) {
       return FALSE;
     }
     return TRUE;